actionpack/test/controller/routing_test.rb

```diff
@@ -1694,7 +1694,7 @@ def test_route_with_subdomain_and_constraints_must_receive_params
     set.draw do
       get "page/:name" => "pages#show", :constraints => lambda { |request|
         name_param = request.params[:name]
-        return true
+        true
```

Comment:
Don't like this rule.

Reply:
(linking for context) It looks like the rule was enabled in 146b1c2e3389bc70ea0b54abf7843fc1d6c8cd5f after #31004 was merged

Reply:
Guess that's why I'd be keen to get Rails under omakase style too.